arch/arm/core/aarch32/cortex_m/tz/secure_entry_functions.ld

```diff
@@ -4,18 +4,24 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/* nRF-specific defines. */
+#ifdef CONFIG_CPU_HAS_NRF_IDAU
+	/* This SOC needs the NSC region to be at the end of an SPU region. */
+	#define NSC_ALIGN \
+		. = ALIGN(CONFIG_NRF_SPU_FLASH_REGION_SIZE) \
+			- (1 << LOG2CEIL(__sg_size))
+	#define NSC_ALIGN_END . = ALIGN(CONFIG_NRF_SPU_FLASH_REGION_SIZE)
+#endif /* CONFIG_CPU_HAS_NRF_IDAU */
+
+
+/* Define NSC_ALIGN */
```

Comment:
what's the point of this comment?

Reply:
To make it clearer that the next block just defines NSC_ALIGN. If it's not clear, I'll remove it.

Reply:
Removed